modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -678,6 +679,7 @@ private JSONArray _getDepotEntriesJSONArray() {
 		return _getDepotEntriesJSONArray(
 			TransformUtil.transform(
 				depotEntryLocalService.getDepotEntries(
```

Comment:
Hi, looking at https://liferay.atlassian.net/browse/LPD-65375 maybe worth to create the `depotEntryService.getDepotEntries(` and use it here.

What do you think?

Reply:
The issue is that service builder is not generating filterFind* methods for depot entries. We'll need to update service builder itself to fix that.